nservicebus/pipeline/steps-stages-connectors_extra_core_[6.0,).partial.md

```diff
@@ -14,6 +14,7 @@ In the diagram User Code can refer to a handler or a saga. If the handler or sag
 
 ### Incoming Pipeline Stages
 
+ * TransportReceiveContext: Behaviors in this stage are responsible signaling failure or success to the transport. This stage provides `ITransportReceiveContext` to it's behaviors.
```

Comment:
Ah ok, sounds good to me. I'd skip that though:
> since the outbox transaction and the batching starts right before the physical message context

as that raises more questions like "what if I don't use outbox", etc. 


> responsible 

responsible for

> it's behaviors.

Should probably be "its behaviors"